tedana/decomposition/eigendecomp.py

```diff
@@ -143,21 +143,22 @@ def tedpca(catd, OCcatd, combmode, mask, t2s, t2sG, stabilize,
             from sklearn.decomposition import PCA
             ppca = PCA(n_components='mle', svd_solver='full')
             ppca.fit(dz)
-            v = ppca.components_
-            s = ppca.explained_variance_
-            u = np.dot(np.dot(dz, v.T), np.diag(1. / s))
+            comp_ts = ppca.components_
+            varex = ppca.explained_variance_
+            voxel_comp_weights = np.dot(np.dot(dz, comp_ts.T), np.diag(1. / varex))
         else:
-            u, s, v = np.linalg.svd(dz, full_matrices=0)
+            voxel_comp_weights, varex, comp_ts = np.linalg.svd(dz, full_matrices=0)
 
         # actual variance explained (normalized)
-        sp = s / s.sum()
-        eigelb = getelbow_mod(sp, return_val=True)
+        varex_norm = varex / varex.sum()
+        eigenvalue_elbow = getelbow_mod(varex_norm, return_val=True)
 
-        spdif = np.abs(np.diff(sp))
+        spdif = np.abs(np.diff(varex_norm))
```

Comment:
Can we change these names (`spdif`, `spdifh`, `spdthr`) to match the others ?

Reply:
I'd be happy to, though I don't know what they are. Any thoughts on what they could be called?

Reply:
Suggested names:
 - `diff_varex_norm` for `spdif` -- it's the difference between normalized, successive singular values
 - `lower_diff_varex_norm` for `spdifh` -- it's the second half of `spdif`
 - `varex_norm_thr` for `spdthr` -- it's a threshold determined by finding the average of the maximum of `spdifh` and the minimum of `spdif`